spec/functions/cidr_to_range_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe 'cidr_to_range' do
```

Comment:
Could you also please add a test case for /32 cidr to ensure it returns empty list.